code/defines/procs/admin.dm

```diff
@@ -68,8 +68,10 @@
 	return .
 
 /proc/key_name_admin(whom)
-	var/message = "[key_name(whom, 1)](<A HREF='?_src_=holder;adminmoreinfo=\ref[whom]'>?</A>)[isAntag(whom) ? "<font color='red'>(A)</font>" : ""][isLivingSSD(whom) ? "<span class='danger'>(SSD!)</span>" : ""] ([admin_jump_link(whom)])"
-	return message
+	if(whom)
+		var/mob/whom_mob = whom //As long as it's not null, will be close enough/has the proc UID() that is all that's needed
```

Comment:
Cast this as whatever the highest level type UID is defined on is. I think that's either atom or datum.